var/spack/repos/builtin/packages/claw/package.py

```diff
@@ -25,20 +26,31 @@ class Claw(CMakePackage):
     version('1.1.0', commit='16b165a443b11b025a77cad830b1280b8c9bcf01', submodules=True)
 
     depends_on('cmake@3.0:', type='build')
+    depends_on('ant@1.9:', type='build')
+    depends_on('bison', type='build')
+    depends_on('flex', type='build')
+
     depends_on('java@8:', when="@2.0:")
     depends_on('java@7:', when="@1.1.0:1.2.3")
-    depends_on('ant@1.9:')
     depends_on('libxml2')
-    depends_on('bison')
 
-    def cmake_args(self):
-        args = []
-        spec = self.spec
+    patch('https://github.com/claw-project/claw-compiler/commit/e9fe6dbd291454ce34dd58f21d102f7f1bdff874.patch',
+          sha256='44a3e17bf6e972db9760fc50bc0948309ee441dab1cdb11816ba675de0138549',
+          when='@:2.0.2%intel')
+
+    # Cache ANT dependencies in the stage directory.
+    # Otherwise, they are cached to the user's home directory.
+    patch('ivy_local_cache.patch')
 
-        args.append('-DOMNI_CONF_OPTION=--with-libxml2={0}'.
-                    format(spec['libxml2'].prefix))
+    # https://github.com/claw-project/claw-compiler/pull/586
+    conflicts('%nag', when='@:2.0.1')
```

Comment:
Can't you have the same patch mechanism than for intel?

Reply:
I thought about it but decided to put a simple conflict since there is actually a release that supports NAG compiler.

Reply:
Ok fine for me.